sdlib/d/gc/arena.d

```diff
@@ -453,6 +530,74 @@ unittest allocLarge {
 	arena.free(&emap, pd3, ptr3);
 }
 
+unittest resizeLarge {
```

Comment:
Try to keep the tests roughly in the same shape the code is in and the whole thing following some logical flow. The test for resize probably shouldn't come before the one to alloc pages.